internal/domain/jaws.go

```diff
@@ -0,0 +1,184 @@
+// MIT License
+//
+// (C) Copyright [2025] Hewlett Packard Enterprise Development LP
+//
+// Permission is hereby granted, free of charge, to any person obtaining a
+// copy of this software and associated documentation files (the "Software"),
+// to deal in the Software without restriction, including without limitation
+// the rights to use, copy, modify, merge, publish, distribute, sublicense,
+// and/or sell copies of the Software, and to permit persons to whom the
+// Software is furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included
+// in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+// THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+// OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+// ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+// OTHER DEALINGS IN THE SOFTWARE.
+
+package domain
+
+import (
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"io"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+
+	pcsmodel "github.com/OpenCHAMI/power-control/v2/internal/model"
+	"github.com/sirupsen/logrus"
+
+	"github.com/OpenCHAMI/power-control/v2/internal/logger"
+	"github.com/hashicorp/go-retryablehttp"
```

Comment:
Which editor(s) are ya'll using? Most of them should have a means of running an import formatter on exit (e.g. [for VS Code](https://hyr.mn/gofmt/)), though you may need to have the stdlib/external/local groups already.

```suggestion
	"github.com/hashicorp/go-retryablehttp"
	"github.com/sirupsen/logrus"

	"github.com/OpenCHAMI/power-control/v2/internal/logger"
	pcsmodel "github.com/OpenCHAMI/power-control/v2/internal/model"
```

Reply:
The formatter that my VS did not sort those as you have them.  I think because I had the broken up into groups already.  Did sort as suggested as that is how they should be.

Reply:
Looks like it's a bit wonky with grouping because goimports is kinda weird (it'll preserve or create additional sections depending on where the out of order entry is) and because VS Code doesn't automatically add a local prefix matching go.mod.

https://github.com/golang/vscode-go/blob/master/docs/settings.md#formattinglocal does, but is weirdly dependent on gopls instead of goimports. I could include it as a workspace setting, though IDK if that's annoying, since I'm not a VS Code user. https://stackoverflow.com/a/57749909 suggests it's probably fine.